src/main/java/org/lisoft/lsml/view_fx/util/WeaponSummary.java

```diff
@@ -286,7 +286,7 @@ private String getSelectorFor(Item aItem) {
     private boolean shouldCountTubes(Item aItem) {
         if (aItem instanceof MissileWeapon) {
             final MissileWeapon missileWeapon = (MissileWeapon) aItem;
-            return missileWeapon.getAmmoPerPerShot() > 1;
+            return missileWeapon.getRoundsPerShot() > 1;
```

Comment:
This is a `MissileWeapon` which is a sub class of `AmmoWeapon`, you can use `getAmmoPerShot()` directly.